source/client.js

```diff
@@ -13,7 +15,11 @@ const pageDetails = parse(window.location.search)
 
 render(
   <div>
```

Comment:
I was wondering when the line would arrive between App component and initial render. This was feeling like it should be a component but ... excuses.
I have refactored out <App />
Improving the meta tags would be nice so happy to chat tomorrow.